src/applications/ask-a-question/form/form.js

```diff
@@ -75,6 +77,11 @@ const formConfig = {
       // saved: 'Your [benefitType] [appType] has been saved.',
     },
   },
+  errorText: () => (
```

Comment:
Again, we mirrored how "CallHRC" is used in platform code. In theory we could clean this up and encapsulate it inside a "CallMyVA311Error" component